src/main/java/edu/kis/powp/jobs2d/builders/Figure1Builder.java

```diff
@@ -0,0 +1,33 @@
+package edu.kis.powp.jobs2d.builders;
+
+import edu.kis.powp.jobs2d.Job2dDriver;
+import edu.kis.powp.jobs2d.drivers.command.ComplexCommand;
+import edu.kis.powp.jobs2d.drivers.command.DriverCommand;
+import edu.kis.powp.jobs2d.drivers.command.OperateToCommand;
+import edu.kis.powp.jobs2d.drivers.command.SetPositionCommand;
+import edu.kis.powp.jobs2d.magicpresets.FiguresJoe;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Figures1Builder implements Job2dDriver, Builder {
+    private final List<DriverCommand> commandList = new ArrayList<>();
+
+    @Override
+    public void setPosition(int x, int y) {
+        this.commandList.add(new SetPositionCommand(x, y));
+    }
+
+    @Override
+    public void operateTo(int x, int y) {
+        this.commandList.add(new OperateToCommand(x, y));
+    }
+
+    @Override
+    public Object build() {
+        if (this.commandList.size() == 0) {
+            FiguresJoe.figureScript1(this);
```

Comment:
Actually, it would be very easy to convert those Builders to a single generic functionality (to cover building command from any script). Actually it is even possible now, because we can externally execute any set of commands on Figures1Builder (but the name doesn't suggest such possibility).